MalwareAnalyzer/views/domain_check.py

```diff
@@ -130,11 +130,11 @@ def get_netloc(url):
     """Get Domain."""
     try:
         domain = ''
-        parsed_uri = urlparse(url)
-        if not urlparse(url).scheme:
+        parse_uri = urlparse(url)
+        if not parse_uri.scheme:
             url = '//' + url
-            parsed_uri = urlparse(url)
-            domain = '{uri.netloc}'.format(uri=parsed_uri)
+            parse_uri = urlparse(url)
+            domain = '{uri.netloc}'.format(uri=parse_uri)
```

Comment:
Is this intent correct?

Reply:
The domain should be available if `scheme` is present as well right?

Reply:
I replace parsed with parse to be consistent with the check_domain function 
you can have domain without scheme and with scheme this is not a problem

Reply:
The variable `domain` is assigned only when scheme is not present.

See: https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/bbce1323ba0b5e6f8bd29ad1b581a47cffa0cda1/MalwareAnalyzer/views/domain_check.py#L137

 I think domain should have indentation of 4 spaces removed.

```
if not parse_uri.scheme:
            url = '//' + url
            parse_uri = urlparse(url)
 domain = '{uri.netloc}'.format(uri=parse_uri)
```

Reply:
simplify the code